proposals/0522-or-patterns.rst

```diff
@@ -1,12 +1,12 @@
 Or Patterns
 ==============
 
-.. author:: David Knothe, Ömer Sinan Ağacan, Sebastian Graf
-.. date-accepted:: 2022-11-29
+.. author:: Sebastian Graf, David Knothe
+.. date-accepted::
```

Comment:
I'm not sure why Ömer's name is being removed from the authors list, and the date-accepted removed, was that deliberate?